### ISSUE TITLE
placeholder text is copied to aria-label, resulting in the text read twice

### ISSUE THREAD

**Issue by du6** [angular/material#10510]
**Actual Behavior**:
 -  The screen reader reads out "Address 2 with hint address 2,  edit text", which reads the placeholder text twice.

 - The reason is that the placeholder text is copied to aria-label if there is no aria-label present (https://github.com/angular/material/blob/master/src/components/input/input.js#L343)

 - By removing the redundant aria-label, the screen reader reads out "Address 2,  edit text", which should be as expected.

**Demo** (or steps to reproduce the issue): *
 - Please listen to the screen reader when the 'Address 2' input get focus in angular material demo page:
https://material.angularjs.org/latest/demo/input

**AngularJS Versions**: *
 -  `AngularJS Version:`1.5.5
 -  `AngularJS Material Version:`1.1.3

----
Shortcut to create a [new CodePen Demo](http://codepen.io/team/AngularMaterial/pen/bEGJdd).
Note: `*` indicates required information. Without this information, your issue may be auto-closed.

> Do not modify the titles or questions. Simply add your responses to the ends of the questions.
  Add more lines if needed.
